src/processing/vCorner/vCorner.xml

```diff
@@ -15,14 +15,18 @@
     <arguments>
         <param desc="Specifies the stem name of ports created by the module." default="vCorner"> name </param>
         <param desc="Sets ouput port to use strict protocol." default="true"> strict </param>
-        <param desc="Number of pixels on the x-axis of the sensor." default="128"> width </param>
-        <param desc="Number of pixels on the y-axis of the sensor." default="128"> height </param>
-        <param desc="Size of Sobel filter used for the detection." default="5"> filterSize </param>
-        <param desc="Length of the temporal window in seconds used to accumulate events." default="0.01"> tempsize </param>
+        <param desc="Number of pixels on the x-axis of the sensor." default="304"> width </param>
```

Comment:
double check that this file isn't actually in the pull request. If it's in the pull request, remove it (make it a new pull request). If it's not just leave this